raiden/storage/utils.py

```diff
@@ -88,9 +88,9 @@ def __getattr__(self, item):
 DB_CREATE_CLIENT = """
 CREATE TABLE IF NOT EXISTS client (
```

Comment:
We could put the type of client here maybe, it could be useful if you want to query data from database and know which type of transport is using that client. Just a thought

Reply:
I have thought about that but i could find any situation in which you want to know the transport, since the transport mode is the transport mode that the hub is running

Reply:
Make sense